projects/canopy/src/lib/quick-action/docs/guide.stories.mdx

```diff
@@ -0,0 +1,87 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Components/Quick Action/Guide" parameters={{ viewMode: 'docs', previewTabs: { canvas: { hidden: true } } }} />
+
+# Quick action
+
+<p class="standfirst">Quick Actions are small buttons that can be used to provide functionality in context, such as editing a form or actions on a card.</p>
+
+## ![example](docs/quick-action/example.png)
+
+## Usage
+
+Quick Actions should be used to increase the affordance of a specific function when a primary CTA would be too prominent. They are typically used to give access to functionality in the context of a piece of content or data. An example would be when allowing the user to change their bank details in the context of showing them bank details:
+
+![usage](usage.png)
+
+Quick actions shouldn’t be used to complete form journeys or to invite the user to read more. In these scenarios, a button or link should be used.
```

Comment:
I think the backtick should be an apostrophe in `shouldn't`?

Reply:
yep, good eye; i'll swap it out and check for others